pkg/apis/radix/v1/radixapptypes.go

```diff
@@ -945,45 +945,50 @@ type RadixVolumeMount struct {
 	Path string `json:"path"` // Path within the pod (replica), where the volume mount has been mounted to
 
 	// GID defines the group ID (number) which will be set as owner of the mounted volume.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
 	GID string `json:"gid,omitempty"` // Optional. Volume mount owner GroupID. Used when drivers do not honor fsGroup securityContext setting. https://github.com/kubernetes-sigs/blob-csi-driver/blob/master/docs/driver-parameters.md
 
 	// UID defines the user ID (number) which will be set as owner of the mounted volume.
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
 	UID string `json:"uid,omitempty"` // Optional. Volume mount owner UserID. Used instead of GID.
 
 	// TODO: describe
 	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
 	SkuName string `json:"skuName,omitempty"` // Available values: Standard_LRS (default), Premium_LRS, Standard_GRS, Standard_RAGRS. https://docs.microsoft.com/en-us/rest/api/storagerp/srp_sku_types
 
 	// TODO: describe
 	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +optional
 	RequestsStorage string `json:"requestsStorage,omitempty"` // Requests resource storage size. Default "1Mi". https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-a-persistentvolumeclaim
 
 	// Access mode from a container to an external storage. ReadOnlyMany (default), ReadWriteOnce, ReadWriteMany.
 	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +kubebuilder:validation:Enum=ReadOnlyMany;ReadWriteOnce;ReadWriteMany;""
 	// +optional
 	AccessMode string `json:"accessMode,omitempty"` // Available values: ReadOnlyMany (default) - read-only by many nodes, ReadWriteOnce - read-write by a single node, ReadWriteMany - read-write by many nodes. https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes
 
 	// Binding mode from a container to an external storage. Immediate (default), WaitForFirstConsumer.
 	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
-	// Deprecated, use BlobFuse2 or AzureFile instead.
+	// Deprecated, use BlobFuse2 instead.
 	// +kubebuilder:validation:Enum=Immediate;WaitForFirstConsumer;""
 	// +optional
```

Comment:
```suggestion
	// Deprecated: use BlobFuse2 instead.
	// Binding mode from a container to an external storage. Immediate (default), WaitForFirstConsumer.
	// More info: https://www.radix.equinor.com/guides/volume-mounts/optional-settings/
	// +kubebuilder:validation:Enum=Immediate;WaitForFirstConsumer;""
	// +optional
```
The first comment must start with "Deprecated: " for field to be flagged as deprecated